src/ViewModel/NextUpViewModel.cpp

```diff
@@ -190,15 +192,16 @@ Song NextUpViewModel::getNextSong() {
         break;
     }
     case PlayMode::ListRepeat: {
-        // TODO
+        song = model.at(0);
```

Comment:
Really working?

Reply:
that's the only one who is right :fireworks: